tests/e2e/v3_curl_test.go

```diff
@@ -274,6 +279,63 @@ func testV3CurlAuth(cx ctlCtx) {
 	}
 }
 
+func testV3CurlAuthWithoutOption(cx ctlCtx) {
```

Comment:
Can we explore the option of modifying the existing `testV3CurlAuth` e2e test to include both w/ and w/o user option cases?